assets/launcher/lang/sklmessages_cs_CZ.properties

```diff
@@ -22,27 +22,27 @@ sidebar.ready.update=Připraven pro aktualizování & hraní
 sidebar.ready.download=Připraven pro stažení & hraní
 
 game.status.play=Hrát
-game.status.play.demo=Hrát Demo
-game.status.play.offline=Hrát Offline
+game.status.play.demo=Hrát demo
+game.status.play.offline=Hrát v offline režimu
```

Comment:
Isn't the previous translation also correct, is it? The new one will not fit in that not too big button.